### DISCUSSION TITLE
Cannot find documentation on controls to change map coloring / legend

### DISCUSSION THREAD

**Discussion by SomeTroglodyte** [endless-sky/endless-sky#9304] [General]
The galaxy map is able to display the legend headings "System:", "You have visited:" and "Government:" which are much more useful than what I usually see. I sometimes see them, but never know why, what changed it. There's no key mappings in the preferences, I can't find documentation, and experimentation only turned up Arrow Up/Down, which set the map to show spam, but cannot restore one of the useful ones.

From source inspection I found another two ways: Hacking the save in a text editor, and set "map coloring" to something negative, or clicking on the left side on one of the spam names... Surprise: Clicking the system or government name just above that reacts meaningfully. The little aside that selecting a planet to land on _always_ resets the coloring to number of outfits on sale (anybody can explain why this is a relevant metric?) - is probably unintentional.

Leaves the question: Is there _any_ predictable way to get the "visited" map (except the "-3" via notepad way)?

* By 'spam' I mean the trade commodities, which only get in the way after you sell your first captured pirate. The new setting in 0.10.3 shows the problem is known, but does not help much, as it blocks a quest and still won't prevent your cargo hold filling up with unsanctioned stuff... Auto-sell on every landing would be better, or a "pick up flotsam only if worth >=10k/ton" setting or somesuch...

**Comment by mOctave** [comment]
Click the field on the left that you want to know about. If you want the "visited" map, click "(not yet visited)" or similar.

https://github.com/endless-sky/endless-sky/assets/73318970/639e3e92-16a1-4e5b-9744-3951a9a5ca9a

**Reply by SomeTroglodyte** [reply]
:+1: Found that later. A little incongruent, "visited" unavailable unless selecting a non-empty system first despite being useful generally, and - ***undocumented***. Now I just have to find out why or when it reverts to count of ships or count of outfits without me doing so...
